repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -14,10 +14,13 @@
 Event = namedtuple('Event', ['action',        # A string representing an event type (see EVENT_TYPES)
                              'in_pkgs',       # A dictionary with packages in format {<pkg_name>: <repository>}
                              'out_pkgs',      # A dictionary with packages in format {<pkg_name>: <repository>}
+                             'from_release',  # A tuple representing a release in format (major, minor)
+                             'to_release',    # A tuple representing a release in format (major, minor)
                              'architectures'  # A list of strings representing architectures
                              ])
 
 EVENT_TYPES = ('Present', 'Removed', 'Deprecated', 'Replaced', 'Split', 'Merged', 'Moved', 'Renamed')
+RELEASES = ((7, 5), (7, 6), (7, 7), (7, 8), (8, 0), (8, 1))  # TODO: bad, bad hardcode
```

Comment:
I am  just curious, why it is needed to hardcode even (O_o) I do not see any reson for that.

Reply:
Well, there's no existing way to get all the releases at the moment.
Technically I could do an additional cycle through all events at the start of `process_events` and accumulate the releases like that, at the cost of total performance and run time. @bocekm told me not to bother with it but I can implement it if you don't want this hardcode there.

Reply:
ug... I don't see any reason for additional cycle. But I haven't read the code so probably your solution is different form the one I have in my mind. There is no time for such changes now, so keep it as it is.